docs/source/index.rst

```diff
@@ -1,6 +1,3 @@
-Home Page
-=========
```

Comment:
Yeah the result of our investigations was that a title is (for some reason) needed to have the bullet point displayed eheh :-(